src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stringModified = sourceString
+    .split(/;\s*/)
+    .filter((value) => value.length)
+    .map((item) => item.split(':').map((el) => el.trim()));
+
+  return stringModified.reduce((prev, next) => {
+    const acc = next[0];
+    const value = next[1];
+
+    prev[acc] = value;
+
+    return prev;
```

Comment:
```suggestion
  return stringModified.reduce((acc, [key, value]) => {
    acc[key] = value;

    return prev;
```
The first argument usually has the name `acc` - accumulator https://www.freecodecamp.org/news/how-to-use-javascript-array-reduce-method/

Reply:
it is strange, that is not my code